src/components/navigation/header/index.tsx

```diff
@@ -103,7 +103,7 @@ const Navigation: FC<NavigationProps> = ({
           margin="auto"
           display="flex"
           justifyContent="space-between"
-          px={{ base: 'sm', xs: 'lg' }}
+          px={{ base: 'sm', xs: 'lg', sm: '2xl' }}
```

Comment:
just to be sure, do we need to keep `xs: 'lg'` ? or only with `base:'sm', sm: '2xl'` are we fine. The ticket description it's not so clear for me.

Reply:
Yes, we need 8, 16, and 24px paddings for those 3 break-points. 
You may find the details in ticket's Figma link: https://www.figma.com/file/8vy6tgk0DV9KSKyJBjiLSH/12.-AS24---Web---Patterns?type=design&node-id=2992%3A27532&mode=design&t=qJZchKdTydbhn0di-1